src/components/Modal/Modal.stories.tsx

```diff
@@ -11,38 +11,32 @@ import Dropdown from '../Dropdown/Dropdown';
 import { Typography } from '../Typography';
 import color from '../../styles/colors';
 
-const hasPositionAbsoluteFix = {
-    transform: 'scale(1)',
-    height: '90vh',
-};
-
 export default {
```

Comment:
Instead of completely deleting this feature. 
Lets give an optional prop that lets the developer decide if he/she wants this or not.

Reply:
This was in the storybook for viewing only
It was used only to fix the absolute view inside storybook

Reply:
ohh cool. 
LGTM 🚀